tests/test_test_runner.py

```diff
@@ -3,48 +3,83 @@
 
 import pytest
 
-from dojo_toolkit.test_runner import DoctestTestRunner, SubprocessTestRunner
+from dojo_toolkit.test_runner import DoctestTestRunner
 
 MODULE_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
 @pytest.fixture
-def code_path():
-    return "/path/to/my/code"
-
-
-class MockTestRunner(SubprocessTestRunner):
-    cmd = "echo"
-
-
-@pytest.fixture
-def mock_test_runner(code_path):
-    return MockTestRunner(code_path, sound_player=mock.Mock())
-
-
-class WrongTestRunner(DoctestTestRunner):
-    cmd = "parangaricutirimicuaro"
-
-
-@pytest.fixture
-def wrong_test_runner(code_path):
-    with mock.patch('dojo_toolkit.test_runner.notifier'):
-        return WrongTestRunner(code_path, sound_player=mock.Mock())
+def code_file(tmpdir):
+    code_file = tmpdir.join('foo.py')
+    return code_file
```

Comment:
Every use of the `code_file` must cast to `str`, what about `return str(code_file)`?